sql/_35_fig_cake/cbrd_24044/cbrd_25214/cases/cbrd_25214.sql

```diff
@@ -0,0 +1,315 @@
+/* Description
+This scenario verified CBRD-25214 issue.
+Remove join predicate already logically evaluated.
+*/
+
+drop table if exists ta;
+drop table if exists tb;
+drop table if exists tp;
+create table ta (cola int, colb int);
+create table tb (cola int, colb int);
+insert into ta values(1,2);
+insert into ta values(2,2);
+insert into ta values(3,2);
+
+insert into tb values(1,2);
+insert into tb values(2,2);
+insert into tb values(3,2);
+insert into tb values(4,2);
+insert into tb values(5,2);
+insert into tb values(6,2);
+
+create index idx_tb on tb(cola,colb);
+create index idx_ta on ta(cola,colb);
+
+create table tp(cola int, colb int ) 
+partition by range(colb)
+(
+    partition b_2 values less than (3),
+    partition b_3 values less than (4) 
+);
+
+insert into tp values(1,2),(2,2),(3,2),(1,3),(2,3),(3,3),(4,3);
+create index idx_tp on tp(cola,colb);
+
+update statistics on ta, tb, tp;
+
+set trace on;
```

Comment:
The developer uses the syntax 'set optimizer 513' and the TC uses the syntax 'set trace on'.
Is there any problem comparing the results?

Reply:
Since our purpose is to verify that unnecessary conditions are deleted, 'trace' is sufficient.